packages/discovery-react-components/src/components/SearchResults/components/Result/Result.tsx

```diff
@@ -20,6 +20,7 @@ import {
   searchResultFooterTitleClass,
   searchResultFooterCollectionNameClass
 } from 'components/SearchResults/cssClasses';
+import { getDocumentTitle } from '../../utils';
```

Comment:
In this case, I'd go with importing with the module path alias `components/SearchResults/utils`.